functions/vpn.bash

```diff
@@ -228,6 +228,8 @@ install_tailscale() {
 ##
 setup_tailscale() {
   local preAuthKey=${preauthkey}
+  # shellcheck disable=SC2154
+  local tags=${tstags}
```

Comment:
We should be double quoting variable assignments whenever possible.

Reply:
not needed here but ok